File: src/main/java/com/bvanseg/gigeresque/common/block/material/Materials.java
package com.bvanseg.gigeresque.common.block.material;

import net.minecraft.block.MapColor;
import net.minecraft.block.Material;

public class Materials {
    public final static Material ACID = new MaterialBuilder(MapColor.TERRACOTTA_GREEN).allowsMovement().lightPassesThrough().notSolid().replaceable().destroyedByPiston().build();
    public final static Material NEST_RESIN = new MaterialBuilder(MapColor.GRAY).burnable().allowsMovement().build();
    public final static Material NEST_RESIN_WEB = new MaterialBuilder(MapColor.GRAY).burnable().allowsMovement().lightPassesThrough().build();
    public final static Material ORGANIC_ALIEN_BLOCK = new MaterialBuilder(MapColor.GRAY).build();
    public final static Material ROUGH_ALIEN_BLOCK = new MaterialBuilder(MapColor.GRAY).build();
    public final static Material SINOUS_ALIEN_BLOCK = new MaterialBuilder(MapColor.GRAY).build();
}

File: src/main/kotlin/com/bvanseg/gigeresque/common/block/material/Materials.kt
package com.bvanseg.gigeresque.common.block.material

import net.minecraft.block.MapColor

/**
 * @author Boston Vanseghi
 */
object Materials {
    val ACID = MaterialBuilder(MapColor.TERRACOTTA_GREEN).allowsMovement().lightPassesThrough().notSolid().replaceable()
        .destroyedByPiston().build()
    val NEST_RESIN = MaterialBuilder(MapColor.GRAY).burnable().allowsMovement().build()
    val NEST_RESIN_WEB = MaterialBuilder(MapColor.GRAY).burnable().allowsMovement().lightPassesThrough().build()
    val ORGANIC_ALIEN_BLOCK = MaterialBuilder(MapColor.GRAY).build()
    val ROUGH_ALIEN_BLOCK = MaterialBuilder(MapColor.GRAY).build()
    val SINOUS_ALIEN_BLOCK = MaterialBuilder(MapColor.GRAY).build()
}